src/web_interface/components/ajax_routes.py

```diff
@@ -162,3 +166,27 @@ def _ajax_get_summary(self, uid, selected_analysis):
             firmware = sc.get_object(uid, analysis_filter=selected_analysis)
             summary_of_included_files = sc.get_summary(firmware, selected_analysis)
         return render_template('summary.html', summary_of_included_files=summary_of_included_files, root_uid=uid, selected_analysis=selected_analysis)
+
+    @roles_accepted(*PRIVILEGES['status'])
+    def _get_asynchronous_stats(self):
```

Comment:
Name does not fit anymore. AJAX (Asynchronous JavaScript and XML) already implies asynchronicity so a name such as `_get_general_stats` fits better